docs/docusaurus/docs/contributing/application_integration_support.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Application support policy
```

Comment:
```suggestion
title: Support policy
```

Reply:
@joshzzheng - support policy for what? You need to specify the type of policy. This isn't a support policy; it's a policy for application integrations, operating systems, and programming languages.